migrations/20181023140057_remove_contact_usernames.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class RemoveContactUsernames extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-abstractmigration-class
+     *
+     * The following commands can be used in this method and Phinx will
+     * automatically reverse them when rolling back:
+     *
+     *    createTable
+     *    renameTable
+     *    addColumn
+     *    renameColumn
+     *    addIndex
+     *    addForeignKey
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function change()
+    {
+        $this->table('contacts')
+            ->changeColumn('contact', 'string', ['null' => true])
```

Comment:
ugh no this should just be the opposite, null out the contact in message , and keep contact null constraint .